package.json

```
@@ -9,7 +9,7 @@
 		"static/src/**/*"
 	],
 	"dependencies": {
-		"@aws-sdk/client-cloudwatch": "3.441.0",
+		"@aws-sdk/client-cloudwatch": "^3.621.0",
```

We may want to keep the pinning?

```suggestion
		"@aws-sdk/client-cloudwatch": "3.621.0",
```

---

Ah yes thanks for the spot. I have pinned the dependency and rerun `yarn`